### PR TITLE
Fix Bug 1180173

### PR DESCRIPTION
**Pull request by KaratekHD**
This should fix https://bugzilla.opensuse.org/show_bug.cgi?id=1180173

### CODE REVIEW

File: package/yast2-alternatives.changes [hunk 1]
```diff
@@ -1,3 +1,8 @@
+-------------------------------------------------------------------
+Thu Dec 17 15:05:54 UTC 2020 - Jens K. <jens@karatek.net>
+
+- Change .desktop file so module launches with xdg-su
```
Comment (imobachgs) [inline package/yast2-alternatives.changes]: Please, add the bug number.

```
 - Change .desktop file so module launches with xdg-su (bsc#1180173).
```

**Review by imobachgs — CHANGES_REQUESTED** [review]
Thanks for the patch. I will backport this change to `SLE-15-SP2` once it is merged.

File: package/yast2-alternatives.changes (hunk 1 above)
Comment (KaratekHD) [reply package/yast2-alternatives.changes]: Done

**Review by imobachgs — APPROVED** [review]
Thanks a lot! I will merge the code and close the bug.

**Pull request merged**